lambdas/db-migration/src/migrations/20201014105102_create_files_table.ts

```diff
@@ -8,9 +8,13 @@ export const up = async (knex: Knex): Promise<void> =>
       .primary();
     table
       .integer('granule_cumulus_id')
+      .notNullable();
```

Comment:
Hmm interesting.      I'm wondering why we didn't set this before.

Reply:
We did. I just moved it around. It was on line 13 previously